impl/client/src/main/javascript/web/pentaho/visual/views/echarts/Abstract.js

```diff
@@ -328,50 +356,107 @@ define([
       });
     },
 
-    _onSelect: function(params) {
-      params.cancelBubble = true;
+    _onSelect: function(echartsEvent) {
       var model = this.model;
-      if(model.isDirty) {
-        return;
-      }
-
-      var filter = null;
-      var srcEvent = params.event.event;
-
-      var filter = this._createFilter(params);
-
+      var filter = this._createFilter(echartsEvent);
       if(filter === null) {
         return;
       }
 
       model.select({
         dataFilter: filter,
-        selectionMode: srcEvent.ctrlKey || srcEvent.metaKey ? "toggle" : "replace"
+        selectionMode: "toggle"
       });
     },
 
-    _createFilter: function(params) {
-      var model = this.model;
-
-      var varsMap = this._getVars(params.data.visualKey);
+    _createFilter: function(echartsEvent) {
+      if(!echartsEvent.hasOwnProperty("data")) {
```

Comment:
Is there a specific case for which this happens? That may be written on a comment?

Reply:
This is needed if user clicks on `axisName`. Only radar uses axisName , for now.